pxtsim/simdriver.ts

```diff
@@ -77,12 +77,22 @@ namespace pxsim {
         public state = SimulatorState.Unloaded;
         public hwdbg: HwDebugger;
         private _dependentEditors: Window[];
+        private _allowedOrigins: Set<string> = new Set();
```

Comment:
is Set supported in all our browser matrix? There are not many origins, you can use a string[] and do linear lookups.

Reply:
Where can I find our browser matrix? Looking at [the Mozilla docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) I see that the Set constructor is supported by browsers. `add()` is supported by all the browsers listed there, Internet Explorer will not return the Set object after add is called. This is fine, since I'm not chaining calls to together.

Reply:
I changed it to use a string[] as suggested.